app/scripts/controllers/dashboard.coffee

```diff
@@ -117,6 +121,22 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
                 resolve maxRatio
               ), reject
 
+        watchCompute 'gpaComputedMax',
+          dependencies: ['allSchools']
+          waitForPromise: true
+          computer: ([allSchools]) -> $q (resolve, reject) ->
+            MIN = 5
+            unless allSchools?
+              resolve MIN
+            else
+              allSchools.then ((schools) ->
+                ratios = schools
```

Comment:
`ratios` should probably be `gpas`, or even just `values`
(also `maxRatio` below)